gci/management/commands/gci_no_issue_tasks.py

```diff
@@ -0,0 +1,29 @@
+import os.path
+
+from ruamel.yaml import YAML
+
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help = 'Fetch GCI data'
+
+    def add_arguments(self, parser):
+        parser.add_argument('output_dir', type=str)
+
+    def handle(self, *args, **options):
+        output_dir = options.get('output_dir')
+
+        tasks_issues = {}
+
+        yaml = YAML()
+
+        with open(os.path.join(output_dir, 'tasks.yaml')) as f:
+            tasks = yaml.load(f)
+
+            for task in tasks:
+                if not tasks[task]['external_url']:
```

Comment:
you need to filter out only task with external_url "which is an issue".  Use IGitt - https://github.com/coala/community/pull/214 and https://github.com/coala/community/pull/215 will give you some tips on how to do this.

Reply:
Omg! I misinterpreted the whole issue. So sorry :disappointed: